packages/flutter/lib/src/gestures/force_press.dart

```diff
@@ -225,6 +235,11 @@ class ForcePressGestureRecognizer extends OneSequenceGestureRecognizer {
       _lastPosition = event.position;
       _lastPressure = pressure;
 
+      // If the device incorrectly reports a pressure outside of pressureMin
+      // and pressureMax, we still want this recognizer to respond normally.
+      if (!pressure.isNaN)
```

Comment:
I'd flip the order of all this a bit. 

debugPrint right away if the raw platform event.pressure isn't within event.pressureMin and event.pressureMax. 

The default _inverseLerp can clamp but the user should be allowed to do whatever they want with `interpolation`.

Then, we hard assert the outcomes of `interpolation`. By that point, pressure needs to be between 0 and 1.

Reply:
What about the case when the result of interpolation is NAN? Do you think that should still be clamped to 0?

Reply:
ah, the current doc already says NaN is ok. I think that's ok in that case (to skip in interpolation returns NaN).